code/obj/item/clothing/shoes.dm

```diff
@@ -381,6 +381,15 @@ ABSTRACT_TYPE(/obj/item/clothing/shoes)
 	AddComponent(/datum/component/wearertargeting/tripsalot, list(SLOT_SHOES))
 	AddComponent(/datum/component/wearertargeting/crayonwalk, list(SLOT_SHOES))
 
+/obj/item/clothing/shoes/moffers
```

Comment:
ideally we should have a way to craft these from mothroachhide i think

Reply:
I was trying to figure out how to do that, but I couldn't find where crafting's done.